### ISSUE TITLE
Adding Alastair Vokachbrodsky to DSVA Slack

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#80404]
# Description

Alastair.Vokach-brodsky@va.gov 

Alastair is a VA 508 office employee who we need to collaborate with sometimes on specialized a11y topics. Recent discussions are about voice control technical standards and guidance. This collaboration would be much easier in slack.

Requesting to add an individual as a:

- Multi-channel guest

What channel(s) do they need to be added to?
- #a11y-champs
- #a11y-specialists (private channel)
- #accessibility-help 
- #accessibility-ops
